install/gcp/deployment_manager/istio-cluster.jinja

```diff
@@ -8,7 +8,6 @@ resources:
     zone: {{ properties['zone'] }}
     cluster:
       name: {{ properties['gkeClusterName'] }}
-      enableKubernetesAlpha: true
       initialClusterVersion: 1.9.2-gke.0
```

Comment:
Should we remove the ```initialClusterVersion``...it'll use the latest
GKE [zonal cluster](https://cloud.google.com/kubernetes-engine/versioning-and-upgrades#versions_available_for_new_cluster_masters)

Reply:
Nerverminds..the default zonal at the moment is ```1.7.12-gke.1``` on GKE so maybe explicitly set it here.  Once even zonal is default on 1.9+, we'll remove this